content/en/docs/reference/config/analysis/ist0110/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: ConflictingSidecarWorkloadSelectors
+layout: analysis-message
+---
+
+This message occurs when more than one Sidecar resource in a namespace selects the same workload instance. This can lead to undefined behavior. See the reference for the [Sidecar](/docs/reference/config/networking/sidecar/) resource for more information.
```

Comment:
```suggestion
This message occurs when more than one `Sidecar` resource in a namespace selects the same workload instance. This can lead to undefined behavior. See the reference for the [Sidecar](/docs/reference/config/networking/sidecar/) resource for more information.
```